### ISSUE TITLE
Error while running the Keras example

### ISSUE THREAD

**Issue by burhanusman** [tf-encrypted/tf-encrypted#638]
So I was running the b_Secure_Classification_with_TFE_Keras_Secure_Model_Serving_  notebook and got the following error: 
`TypeError: run() got an unexpected keyword argument 'tag` `

    TypeError                                 Traceback (most recent call last)
    <ipython-input-12-ad867501d4fb> in <module>
      9     sess,
     10     num_steps=3,
    ---> 11     step_fn=step_fn)

    ~\AppData\Local\Continuum\anaconda3\lib\site-packages\tf_encrypted\serving\queues.py in 
    run(self, sess, num_steps, step_fn)
     55     if num_steps is not None:
     56       for _ in range(num_steps):
    ---> 57         self.run_step(sess)
     58         if step_fn is not None:
     59           step_fn()

    ~\AppData\Local\Continuum\anaconda3\lib\site-packages\tf_encrypted\serving\queues.py in 
    run_step(self, sess, tag)
     45     Serve single computation.
     46     """
    ---> 47     sess.run(self.step_op, tag=tag)
     48 
     49   def run(self, sess, num_steps=None, step_fn=None):

Any help would be really appreciated!

**Comment by jvmancuso** [comment]
Hi @burhanusman, can you provide some environment information?  OS, and version numbers for all of syft, tensorflow, and tf-encrypted would be helpful to get to the bottom of this one!

Also, it looks like you may have modified the notebook, so if you can share your code that would be great as well!

**Comment by burhanusman** [comment]
Hi @jvmancuso, thanks for looking into this.

I'm using Windows 10. 
Syft is not installed.
tenserflow - 1.14.0
tf-encrypted - 0.5.7

I haven't modified the notebook, I was running it as it is on Github. 

So it seems like the session.run() doesn't have 'tag' as a parameter, but we are passing 'tag' while calling it.

**Comment by jvmancuso** [comment]
Good catch! Sorry, thought you were referring to the PySyft notebooks in the Udacity course. This will be fixed once #639 is merged/released.  In the meantime, you can patch it yourself by changing the line
```python
from tensorflow.keras import backend as K
```
from cell 1 to
```python
from tf_encrypted.keras import backend as K
```
